src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java

```diff
@@ -1559,19 +1790,21 @@ private void hitBlockWithToolInHand(@Nullable final BlockPos blockPos, final boo
             if (!world.isRemote)
             {
                 MineColonies.getNetwork().sendToAllAround(
-                  //todo: correct side
-                  new BlockParticleEffectMessage(blockPos, world.getBlockState(blockPos), 1),
-                  new NetworkRegistry.TargetPoint(world.provider.getDimension(), blockPos.getX(), blockPos.getY(), blockPos.getZ(), BLOCK_BREAK_PARTICLE_RANGE));
+                        //todo: correct side
+                        new BlockParticleEffectMessage(blockPos, world.getBlockState(blockPos), 1),
+                        new NetworkRegistry.TargetPoint(world.provider.getDimension(), blockPos.getX(), blockPos.getY(), blockPos.getZ(), BLOCK_BREAK_PARTICLE_RANGE));
             }
             world.playSound((EntityPlayer) null,
-              blockPos,
-              block.getSoundType(blockState, world, blockPos, this).getBreakSound(),
-              SoundCategory.BLOCKS,
-              block.getSoundType(blockState, world, blockPos, this).getVolume(),
-              block.getSoundType(blockState, world, blockPos, this).getPitch());
+                    blockPos,
+                    block.getSoundType(blockState, world, blockPos, this).getBreakSound(),
+                    SoundCategory.BLOCKS,
+                    block.getSoundType(blockState, world, blockPos, this).getVolume(),
+                    block.getSoundType(blockState, world, blockPos, this).getPitch());
         }
     }
 
+    //todo resolve problem if citizen don't get to replentish at home.
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)